styles/main.css

```diff
@@ -41,7 +41,12 @@ h3 {
 }
 a {
   color: #6b9000;
+  text-decoration: none;
+}
+a:hover {
+  text-decoration: underline;
 }
+
```

Comment:
Make the default for ALL links such that they are NOT underlined until they're hovered, then override this in the specific cases where you need to deviate from the default